nature_importer.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+import argparse
+import csv
+import os
+import importer_utils as utils
+import wikidataStuff.wdqsLookup as lookup
+from NatureArea import NatureArea
+from PreviewTable import PreviewTable
+from Uploader import Uploader
+
+DATA_DIRECTORY = "data"
+reserves_file = "NR_polygon.csv"
+nationalparks_file = "NP_polygon.csv"
+edit_summary = "test"
+
+
+def get_data_from_csv_file(filename):
```

Comment:
Since you use almost the same thing in `reserve_harvester.py` I would recommend moving it to utils instead.